index.js

```diff
@@ -179,7 +180,7 @@ module.exports = {
 
     var ui = this.ui;
 
-    return jshintTrees(tree, {
+    return jshintTrees(new Funnel(tree, { include: [/\.js$/] }), {
```

Comment:
PR that introduced this: https://github.com/joliss/node-walk-sync/pull/5

relevant code-points:

* https://github.com/broccolijs/broccoli-funnel/blob/master/index.js#L230-L234
* https://github.com/joliss/node-walk-sync/blob/master/index.js#L53
* https://github.com/stefanpenner/matcher-collection/blob/master/test.js